### ISSUE TITLE
org-gcal-fetch forces org files into active window (fix included)

### ISSUE THREAD

**Issue by alanconway** [kidd/org-gcal.el#202]
The problem is
https://github.com/kidd/org-gcal.el/blob/396a30cf256c8b32a69fd73a081364937b43a1c7/org-generic-id.el#L242
The call `switch-to-buffer` should be `set-buffer`. To quote the emacs source:
```
(defun switch-to-buffer (buffer-or-name &optional norecord force-same-window)
  "Display buffer BUFFER-OR-NAME in the selected window.

WARNING: This is NOT the way to work on another buffer temporarily
within a Lisp program!  Use `set-buffer' instead.  That avoids
messing with the window-buffer correspondences.
```

**Comment by beltrame** [comment]
I can confirm the issue and that the fix works. Perhaps @alanconway should make a PR and link it here?

**Comment by alanconway** [comment]
PR https://github.com/kidd/org-gcal.el/pull/205

**Comment by telotortium** [comment]
Resolved by #205
